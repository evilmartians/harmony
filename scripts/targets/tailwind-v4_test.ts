import { buildTailwindv4Palette } from "./tailwind-v4.ts";
import { testExportTarget } from "./testUtils.ts";

Deno.test("Tailwind v4 palette theme override", async (t) => {
  await testExportTarget(t, buildTailwindv4Palette, [
    "index.css",
  ]);
});

