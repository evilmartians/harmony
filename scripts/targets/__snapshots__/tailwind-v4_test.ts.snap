export const snapshot = {};

snapshot[`Tailwind v4 palette theme override 1`] = `
"
@theme {
  --color-*: initial;

  --color-white: #fff;
  --color-black: #000;

  --color-red-100:oklch(0.966797 0.0171875 20);
  --color-red-500:oklch(0.742188 0.151562 20);
  
  --color-orange-100:oklch(0.966797 0.0171875 43.3333);
  
}"
`;
